### PR TITLE
Disable CURSOR  on shard for FT.PROFILE

### PR DESCRIPTION
**Pull request by ashtul**
With Coordinator, WITHCURSOR is used internally.

### CODE REVIEW

File: src/aggregate/aggregate_exec.c [hunk 1]
```diff
@@ -286,14 +286,24 @@ static int buildRequest(RedisModuleCtx *ctx, RedisModuleString **argv, int argc,
 #define PROFILE_FULL 1
 #define PROFILE_LIMITED 2
 
-static void parseProfile(AREQ *r, int withProfile) {
+static int parseProfile(AREQ *r, RedisModuleCtx *ctx, int withProfile, RedisModuleString **argv, int argc, QueryError *status) {
```
Comment (oshadmi) [inline src/aggregate/aggregate_exec.c:289]: Unused argument `RedisModuleCtx *ctx`

File: tests/pytests/test_profile.py [hunk 2]
```diff
@@ -150,44 +150,9 @@ def testProfileAggregate(env):
...
-  # test second result
-  actual_res = conn.execute_command('ft.cursor', 'read', 'idx', actual_res[1])
-  env.assertEqual(actual_res[0], [1L, ['t', 'world']])
-  env.assertGreater(actual_res[1], 0)
-  env.assertEqual(actual_res[2], None)
-
-  # test final result with profile
-  actual_res = conn.execute_command('ft.cursor', 'read', 'idx', actual_res[1])
-  env.assertEqual(actual_res[2], expected_res)
+  env.expect('ft.profile', 'idx', 'aggregate', 'query', '*', 'WITHCURSOR').error().contains('FT.PROFILE does not support cursor')
```
Comment (oshadmi) [inline tests/pytests/test_profile.py:155]: We do not allow "external" use of cursor with ft.profile - OK
How can we test "internal" use of cursor with ft.profile when `RS_COORDINATOR` is defined?

**Review by oshadmi — APPROVED** [review]

File: tests/pytests/test_profile.py (hunk 2 above)
Comment (ashtul) [reply tests/pytests/test_profile.py:155]: On the coordinator, if it wasn't enabled, the profile tests would have failed.

**Comment by ashtul** [comment]
Related to [commit](https://github.com/RediSearch/RSCoordinator/pull/245/commits/cbc8f37503b3343d3608830121981ebc18c198ae)

**Review by oshadmi — APPROVED** [review]
Thanks for the discussion - we will create an issue to sanity test cursor on Profile in Coordinator.

**Pull request merged**